easybuild/easyconfigs/f/FIt-SNE/FIt-SNE-1.1.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,37 @@
+# easybuild easyconfig
+# John Dey jfdey@fredhutch.org  fizwit@github
+easyblock = 'CmdCp'
+
+name = 'FIt-SNE'
+version = '1.1.0'
+
+homepage = 'https://github.com/KlugerLab/FIt-SNE'
+description = """t-distributed stochastic neighbor embedding (t-SNE) is widely used for
+ visualizing single-cell RNA-sequencing (scRNA-seq) data, but it scales poorly to large
+ datasets. We dramatically accelerate t-SNE, obviating the need for data downsampling, and
+ hence allowing visualization of rare cell populations. Furthermore, we implement a
+ heatmap-style visualization for scRNA-seq based on one-dimensional t-SNE for simultaneously
+ visualizing the expression patterns of thousands of genes."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['https://github.com/KlugerLab/FIt-SNE/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9a985fe1bb1e9c17fff0e81968e794eb834ccf002695d02294f2ff9ac564b873']
+
+dependencies = [
+    ('FFTW', '3.3.8', '-gompi-2018b', True),
```

Comment:
I can't see how this could possibly work well. Why not use the gompi-2018b as the toolchain?

Reply:
good idea, I will update the toolchain.

Reply:
After moving it to gompi-2018b, you should only need to specify `('FFTW', '3.3.8')`

Reply:
fixed